src/azure-cli/azure/cli/command_modules/managedservices/custom.py

```diff
@@ -0,0 +1,219 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import uuid
+from knack.log import get_logger
+from msrestazure.tools import parse_resource_id, is_valid_resource_id
+
+logger = get_logger(__name__)
+default_api_version = '2018-06-01-preview'
```

Comment:
Is the API version for Managed Services, or an input for the API version of the interfacing resources?

Reply:
This is the API version for managed services.

Reply:
In that case, please remove. If you want your module to work with multiple APIs, there are several things you would need to do differently, and this is not the mechanism the CLI allows for that.

Reply:
@tjprescott could you point to an example where api version is handled correctly? thank you.

Reply:
I chatted with Travis offline regarding the API version. For now, I have created an issue to use a hardcoded value in the commands. I am going to follow up Laurent regarding the Python SDK.